cfme/utils/appliance/__init__.py

```diff
@@ -439,7 +440,13 @@ def name(self):
 
     @property
     def server(self):
-        return self.collections.servers.get_master()
+        sid = self._rest_api_server().id
+        return self.collections.servers.instantiate(sid=sid)
```

Comment:
Question: Why are you not using the `get_master` function?

Reply:
Please read the commit messages.

>  In replicated environment, we can have two masters. In such case one of
them (possibly incorrect one) was selected by the tests. This PR
tries to select the the correct rest-api object matching the appliance
in the context.

Reply:
I think this is an important thing to document in-line, not in the commit message.

A docblock on _rest_api_server  would go a long way to making the intent of this pattern clear to any contributor.

Reply:
OK. I am adding the doc to `get_masters`